src/sentry_webhook/test.action.sh

```diff
@@ -19,6 +19,8 @@ rm -rf "${TEST_DIR}"
 
 # Use commonjs modules, jasmine runs in node.
 tsc -p "${ROOT_DIR}/src/sentry_webhook" --outDir "${TEST_DIR}" --module commonjs
+
+export NODE_OPTIONS=--openssl-legacy-provider
```

Comment:
Why is this needed? It's a very intrusive change, touching all our entrypoints.

Reply:
It looks like that forces a very old OpenSSL version, which does not even support TLS1.3
What are the offending dependencies?
Can we make sure we use more recent OpenSSL? (Looks like v3?)